screen2.0/src/app/downloads/data-matrices.tsx

```diff
@@ -0,0 +1,497 @@
+              <Grid2 xs={8}>
+                <Typography mt="auto" variant="h5">Human</Typography>
+                <Divider />
+                <Typography variant="subtitle2">2,348,854 cCREs</Typography>
+                <Typography variant="subtitle2">1,678 cell types</Typography>
+              </Grid2>
+              <Grid2 xs={4}>
+                <Image src={Human} alt={"Human Icon"} width={50} />
+              </Grid2>
+              <Grid2 xs={12}>
+                {selectorButton({ assembly: "Human", assay: "DNase" })}
+                {selectorButton({ assembly: "Human", assay: "H3K4me3" })}
+                {selectorButton({ assembly: "Human", assay: "H3K27ac" })}
+                {selectorButton({ assembly: "Human", assay: "CTCF" })}
+              </Grid2>
+              <Grid2 xs={8}>
+                <Typography variant="h5">Mouse</Typography>
+                <Divider />
+                <Typography variant="subtitle2">926,843 cCREs</Typography>
+                <Typography variant="subtitle2">366 cell types</Typography>
+              </Grid2>
+              <Grid2 xs={4}>
+                <Image src={Mouse} alt={"Mouse Icon"} width={50} />
+              </Grid2>
+              <Grid2 xs={12}>
+                {selectorButton({ assembly: "Mouse", assay: "DNase" })}
+                {selectorButton({ assembly: "Mouse", assay: "H3K4me3" })}
+                {selectorButton({ assembly: "Mouse", assay: "H3K27ac" })}
+                {selectorButton({ assembly: "Mouse", assay: "CTCF" })}
+              </Grid2>
+            </Grid2>
+            <Grid2 container xs={9.5}>
+              <Grid2 xs={4}>
+                <Button
+                  variant="outlined"
+                  fullWidth
+                  onClick={() => null}
+                  endIcon={<Download />}
+                  sx={{ mr: 1, mb: 1, mt: 3, textTransform: "none" }}
+                  href={matrixDownloadURL(selectedAssay, "signal")}
+                >
+                  {`${selectedAssay.assay === "DNase" ? "Read-Depth Normalized Signal Matrix" : "Fold-Change Signal Matrix"}`}
+                </Button>
+                <Button
+                  variant="outlined"
+                  fullWidth
+                  endIcon={<Download />}
+                  sx={{ textTransform: "none", mb: 1 }}
+                  href={matrixDownloadURL(selectedAssay, "zScore")}
+                >
+                  Z-Score Matrix
+                </Button>
+                <Autocomplete
+                  sx={{ mb: 3 }}
+                  disablePortal
+                  id="combo-box-demo"
+                  options={fData}
+                  renderInput={(params) => <TextField {...params} label={"Search for a Biosample"} />}
+                  getOptionLabel={(biosample: BiosampleUMAP) => biosample.displayname + " — Exp ID: " + biosample.experimentAccession}
+                  blurOnSelect
+                  onChange={(event, value: any) => setSearched(value)}
```

Comment:
since event is not being used here, it can be replaced with _,  onChange={(_, value: any) => setSearched(value)}